### ISSUE TITLE
Localization

### ISSUE THREAD

**Issue by brianloveswords** [openbadges/openbadges-specification#1]
[tracking mozilla/openbadges-discussion#2]

**Comment by ottonomy** [comment]
:+1: to the next version of the OBI implementing language. :+1: to using JSON-LD's `@language` tag.

**Comment by barthanssens** [comment]
+1 , localization and the possibility to describe badges in multiple languages is very important in a lot of countries (e.g. Belgium 3 official languages, South-Africa has 11 official languagesà

**Comment by timothyfcook** [comment]
This item has been added to the Badge Alliance roadmap for Q3-Q4 in 2016: https://app.roadmunk.com/publish/aac4dc4e31eedfe37ac85a0a667d2e00655a455d

**Comment by ottonomy** [comment]
Converting this issue to 2.0-recommendation status.

##  Defining a Badge Object in Multiple Languages

Issuers (publishers of badge objects) should be able to offer badge resources in multiple languages and to specify the language an item is published in.

**User Stories**:

1. As an Issuer who wants to issue badges in a number of languages, I would like to publish my issuer information in multiple languages, so that diverse audiences can easily understand my organization.
2. As an Issuer, I'd like to publish translations of my BadgeClasses in multiple languages, so that diverse audiences can easily understand the achievements I am recognizing.
3. As a badge consumer, I'd like to discover badge issuer profiles, classes, and assertions that are published in my language so that I can easily access resources I understand.
4. As a badge consumer, I'd like to be see what language badge resources are published in and, if they are not published in my preferred language, access machine-generated translations in my preferred language.
5. As a developer of badge-related applications, I would like to use a JSON-LD context file specific to my language, so I can quickly understand all the properties of badge objects.

**Comment by ottonomy** [comment]
Prototype stub of a Badge Object identifying another Badge Object that is the same as the present one but published in a different language. 

```jsonld
{
  "@context": "https://w3id.org/openbadges/v2",
  "@language": "fr-fr",
  "@id": "https://iss.org/1?lang=fr-fr"
  "sameAs": [
    {
      "@id": "https://iss.org/1?lang=fi-fi",
      "@language": "fi-fi"
    }
  ]
}
```

**Comment by ottonomy** [comment]
Another approach: Include multiple values for each property in multiple languages:

```jsonld
{
  "@context": "https://w3id.org/openbadges/v2",
  "@id": "https://iss.org/1"
  "name": [
    {"@language": "fi-fi", @value: "osaamismerkki"},
    {"@language": "en-us", "@value": "open badge"}
  ]
}
```

I think I like the multiple documents approach better.

**Comment by sznattila** [comment]
User story:
1. As an issuer I have issued an assertion 2 months ago in English. Now I want to "update" the issued assertions so it would contain new languages and the consumers and displayers can view the assertion in several languages.
2. As a consumer I want to provide a "translation" for my assertion in 3 other languages before the assertion is baked and issued to me. - how the translation will be verified?
3. As a consumer I want to provide a "translation" for my assertion in 3 other languages after the assertion is already baked and issued to me. - how the translation will be verified and baked into the assertion?

**Comment by mgylling** [comment]
Closing this issue, as it was resolved by the 2.0 Recommendation. This issue and all other issues resolved by 2.0 can be found via the 2.0 Release Candidate milestone and PR #99.
